metadata/src/test/java/org/apache/kafka/controller/QuorumFeaturesTest.java

```diff
@@ -58,10 +61,13 @@ public void testDefaultFeatureMap() {
             MetadataVersion.MINIMUM_KRAFT_VERSION.featureLevel(),
             MetadataVersion.LATEST_PRODUCTION.featureLevel()));
         for (Features feature : Features.PRODUCTION_FEATURES) {
-            expectedFeatures.put(feature.featureName(), VersionRange.of(
-                0,
-                feature.defaultValue(MetadataVersion.LATEST_PRODUCTION)
-            ));
+            short maxVersion = feature.defaultValue(MetadataVersion.LATEST_PRODUCTION);
+            if (maxVersion > 0) {
+                expectedFeatures.put(feature.featureName(), VersionRange.of(
+                    feature.minimumProduction(),
+                    maxVersion
+                ));
```

Comment:
The code in the test is pretty much the same as the actual code. Does this and the next test add much value?